docs/reference-architectures/hybrid-networking/vnet-topology-hub-spoke.md

```diff
@@ -0,0 +1,430 @@
+---
+title: Implementing a hub-spoke topology in Azure
+description: >-
+  How to implement a hub-spoke network topoly in Azure.
+author: telmosampaio
+ms.service: guidance
+ms.topic: article
+ms.date: 05/05/2017
+ms.author: pnp
+
+pnp.series.title: Implement a hub-spoke network topology in Azure
+pnp.series.prev: expressroute
+cardTitle: Improving availability
+---
+# Implement a hub-spoke topology in Azure
+
+This reference architecture shows how to implement a hub-spoke topology in Azure, using the hub as a gateway between Azure and your on-premises datacenter.
+
+Traffic flows between the on-premises datacenter and Azure through an ExpressRoute or VPN gateway connection. This connection is made to a hub virtual network (VNet) in Azure, which in turn is peered to other VNets in the same Azure region, as seen in the piture below.[**Deploy this solution**](#deploy-the-solution).
+
+> [!NOTE]
+> Azure has two different deployment models: [Resource Manager](/azure/azure-resource-manager/resource-group-overview) and classic. This reference architecture uses Resource Manager, which Microsoft recommends for new deployments.
+
+Typical uses for this architecture include:
+
+* Workloads deployed in different environments (development, testing, production) that require the specific services that can be shared (in the hub VNet) while maintaining isolation (deployed to spokes).
+* Workloads that do not require connectivity among themselves, but require access to shared services, such as DNS, AD DS, firewall, among others.
+* Enterprises that require a central control over security aspects (firewall in the hub as a DMZ), and segregated management for each workload (individual spokes).
+
+## Architecture
+
+The following diagram highlights the important components in this architecture:
+
+> A Visio document that includes this architecture diagram is available for download from the [Microsoft download center][visio-download]. This diagram is on the "Hub Spoke" page.
+
+![[0]][0]
+
+The hub VNet, and each spoke VNet, can be implemented in different resource groups, and even different subscriptions, as long as they belong to the same Azure tenant in the same Azure region. This allows for a decentralized management of each workload, while sharing services maintained in the hub VNet. However, if you have several spokes that need to connect to the same hub, you will run out of possible peering connections very quickly due to the [limitation on number of VNets peerings per VNet][vnet-peering-limit].
```

Comment:
Up to 10, upgraded to 50 with a support call, max of 100 once PG gets involved. I removed that sentence, again, since we cover that later on.